docs/api/model_selection/validation_curve.rst

```diff
@@ -45,7 +45,7 @@ In the next visualizer, we will see an example that more dramatically visualizes
 
 .. code:: python
 
-    from sklearn.svc import SVM
+    from sklearn.svm import SVC
```

Comment:
Fixed incorrect import statement `from sklearn.svc import SVM` to `sklearn.svm import SVC`